automata/fa/nfa.py

```diff
@@ -9,7 +9,7 @@
 
 import automata.base.exceptions as exceptions
 import automata.fa.fa as fa
-from automata.regex.parser import parse_regex
+from automata.regex.parser import parse_regex, RESERVED_CHARACTERS
```

Comment:
@eliotwrobson Kudos for making this a constant rather than inlining in the below code—it's probably better that way.